### ISSUE TITLE
JSONDecodeError: Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by azataiot** [emreozcan/tdk-py#2]
**Describe the bug**
JSONDecodeError: Expecting value: line 1 column 1 (char 0) while running the example `tdk.gts.suggest("feldispat")` on Github.

**To Reproduce**
1. `tdk.gts.suggest("ispat")`

**Expected behavior**

**Screenshots**
![CleanShot 2022-06-12 at 20 37 22](https://user-images.githubusercontent.com/8280770/173245906-e3e2adce-110a-43e3-816b-db51349b3027.png)

**Environment (please complete the following information):**
 - OS: Google Colab
 - Python Version python3.7
 - tdk-py Version tdk-py==1.1.1

**Additional context**

**Comment by emreozcan** [comment]
Hello! Thanks for your interest, and greetings to Kazakhstan!

I can confirm the issue exists:
![](https://user-images.githubusercontent.com/47896566/173247496-f02a8a06-a093-4e6e-9ea8-6d661f7c780f.png)

Looking for causes/solutions!

**Comment by emreozcan** [comment]
The cause is TDK returning a 0-byte response on suggestion queries.
Test for yourself: https://sozluk.gov.tr/oneri?soz=feldispat

I will add a new exception `TdkSuggestUnexpectedResponseException` inheriting from `TdkUnexpectedResponseException`, and have the function raise that exception in this case.

# Alternative
You can use the `difflib` module from the standard library to fill this gap.

```python
from difflib import get_close_matches

import tdk.gts

_autocomplete = tdk.gts.index()  # You probably want to store this in a variable because it is slow to get.
print(get_close_matches("feldispat", _autocomplete))
# This will print: ['feldspat', 'ispat', 'fesat']
```

**Comment by emreozcan** [comment]
The changes are implemented in the [new version (1.2.0)](https://pypi.org/project/tdk-py/1.2.0/)!

`pip install --upgrade tdk-py` should do it!

Сәттілік тілейміз, Түркиядан Қазақстанға сәлем!

**Comment by azataiot** [comment]
Thanks a lot for your work and effort. 

çok teşekkürler!
